modules/service/import-service.js

```diff
@@ -113,9 +114,8 @@ class ImportService {
         document.datasetHeader = metadata.datasetHeader;
         document.signature = metadata.signature;
 
-
-        ImportUtilities.sortStringifyDataset(document);
-        return document;
+        // todo add otJsonService
```

Comment:
remove comment

Reply:
This is a placeholder for new sorting method which should be implemented in the next sprint